src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -95,19 +125,49 @@ public long claimedArea(UUID playerUUID) {
                 .sum();
     }
 
+    @StringProvider(
+            text = "Claim Block",
+            description = "If Eco is being used, the Blocks player can buy will be showed,\n " +
+                    "or will show like [initial + Bonus + Accrued + Reaming]",
+            iconName = "map",
+            iconColor = Color.BLUE_GREY,
+            iconFamily = Family.REGULAR,
+            showInPlayerTable = true
+    )
+    public String claimBlock(UUID playerUUID) {
+        if (api.isEconomyModeEnabled()) {
+            return "Remaining Blocks: " + getPlayerData(playerUUID).getRemainingClaimBlocks();
+        }
+        return getPlayerData(playerUUID).getInitialClaimBlocks() + "+" +
+                getPlayerData(playerUUID).getBonusClaimBlocks() + "+" +
+                getPlayerData(playerUUID).getAccruedClaimBlocks() + "+" +
```

Comment:
Consider creating a separate StringProvider for each variable - Relying on the description to explain what the numbers mean can be confusing to users